main.tf

```diff
@@ -1,7 +1,8 @@
 module "example1" {
   source = "./modules/Talisker"
   monitorName = "Talisker NRDB Alerts"
-  frequency = 15
+  period = "EVERY_15_MINUTES" # period as specified by resouce
```

Comment:
The new resource takes an enum value here so ive added this new parm in addition to the numeric one (as thats still used for calculations)